src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected int status;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.status = 0;
```

Comment:
Status does not exactly describe what it is for.
From what I gather, it may be redundant as isDone seem to perform its job.